docs/videos/healthcare-and-life-sciences/Factor-Xa-example/submit.sh

```diff
@@ -7,9 +7,10 @@
 # Creative Commons Attribution 4.0 International (CC BY 4.0) License.
 # See <https://creativecommons.org/licenses/by/4.0/>.
 
-#SBATCH -N 2
```

Comment:
Here I am wondering if there would not be value to keep both submit scripts, cpu and gpu, perhaps naming them submit_cpu, submit_gpu? but then, I suppose it would be hard to reconcile the blueprint to support both, right?

Reply:
I think the main reason to only have 1 is to reduce complexity of maintenance. From preliminary testing, the simulation is cheaper and faster on GPUs. I did leave both cpu and gpu for the water script as I think it is important to at least have 1 example showing both.

Also, since this is git if we really want to dig out the cpu solution then we can don't lose that option.